ouster_ros/metadata/os0_64.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
Lets leave sensor specific files in the robot config repositories instead of having them in the main driver?